package.json

```diff
@@ -22,6 +22,7 @@
     "download-akzidenz": "lerna exec -- node ../../scripts/download-akzidenz.js",
     "electron-rebuild": "npm run electron-rebuild --workspace mongodb-compass",
     "node-rebuild": "node ./scripts/rebuild.js kerberos keytar interruptor",
+    "prepackages-publish": "npm install",
```

Comment:
Should we make this `npm ci` and just error if something is off?

Reply:
Using `ci` here will have a downside of slowing down the publish process significantly, I'd prefer to avoid that. But if you feel like it's important to do `ci` instead of `install` here, sure, let's change it

Reply:
Ah right .. cause `ci` would re-install all the node modules ... :/

It would be good if it doesn't change the `package-lock.json` right before publishing, but yeah we still want to move this to CI and try to use a package-lock in `hadron-build release` so npm install should be ok! ..

If a package json is changed it would be added / committed and pushed too?

Reply:
Oh, that's a good point actually. ~So at the point we are publishing, all changes are already commited and even when changed will not get into commit history. Saying that I think lerna will throw if you would try to publish with git history that is not clean~

Okay, so everything I typed above is true, but I just went to double-check what is the flow of versioning-publishing and remembered that we already do an install on `version` to make package-lock include all latest changes. The only thing we are not doing at the moment is actually install the dependencies (to speed up the process):

https://github.com/mongodb-js/compass/blob/f9eeff32620075d8c0961572ae541ccceee89502/scripts/bump-private-dependencies.js#L62

So maybe in this PR I should just drop the `--package-lock-only` flag from this command, does this makes sense?

Reply:
yep i think so